client/analytics/report/stock/table.js

```diff
@@ -155,19 +155,19 @@ class StockReportTable extends Component {
 				value: formatValue( currency, 'number', products ),
 			},
 			{
-				label: __( 'out of stock', outofstock, 'woocommerce-admin' ),
+				label: __( 'out of stock', 'woocommerce-admin' ),
 				value: formatValue( currency, 'number', outofstock ),
 			},
 			{
-				label: __( 'low stock', lowstock, 'woocommerce-admin' ),
+				label: __( 'low stock', 'woocommerce-admin' ),
 				value: formatValue( 'currency, number', lowstock ),
```

Comment:
I just noticed, this line.. looks like a bug 😄 this should be `'currency', 'number'` right? I don't know how this works because looking at the function it would likely only return null or undefined based on these inputs: https://github.com/woocommerce/woocommerce-admin/blob/main/packages/number/src/index.js#L49

Reply:
Yikes! Good catch. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/4980/commits/a8cdc5993ec6f42d14f48b4379a21aa4e89e7779